### ISSUE TITLE
ADA Violation - roles and parent roles

### ISSUE THREAD

**Issue by david-kirkland** [angular-ui/bootstrap#6169]
### Bug description:
Certain roles must be contained by particular parent roles in order to perform the intended accessibility functions.  The `<td>`s used for days in the datepicker have `role="gridcell"`.  This role needs a parent role... `role="row"`.  The absence of this parent role results in a WCAG2 (A) violation.

### Reproduce the issue:
This FireFox plugin will allow you to test accessibility...
https://addons.mozilla.org/en-us/firefox/addon/axe-devtools/

_Make sure the datepicker is visible when you analyze the page._

### Solution:
You just need to add `role="row"` to the `<tr>`.  Example...

`<tr class="uib-weeks" ng-repeat="row in rows track by $index" role="row">`

line 14 ... `bootstrap/template/datepicker/day.html`
line 10 ... `bootstrap/template/datepicker/month.html`
line 10 ... `bootstrap/template/datepicker/year.html`

### Version of Angular, UIBS, and Bootstrap:
Angular:  1.5.2
UIBS:  1.2.5
Bootstrap:  3.3.1

**Comment by wesleycho** [comment]
Up for filing a PR?

**Comment by david-kirkland** [comment]
Done... [https://github.com/angular-ui/bootstrap/pull/6170](url)
